### ISSUE TITLE
reqwest problem disabling wax.leaf.js tile and grid loading in IE9

### ISSUE THREAD

**Issue by mikejcorey** [cutting-room-floor/wax#264]
I believe this is actually a problem in the underlying reqwest code, but it breaks the loading of tiles and grids in IE 9 under some circumstances.

This works the first time a page is loaded, but the callback function returns no data once the page is reloaded:

var tilejsonurl = 'path/to/tile/jsonp';

wax.tilejson(tilejsonurl, function(tilejson) {
    var wax_layer = new wax.leaf.connector(tilejson);  
    wax_layer.addTo(self.map);
});

This can be fixed by commenting out the following in reqwest.js:

if (typeof script.onreadystatechange !== 'undefined') {
    // need this for IE due to out-of-order onreadystatechange(), binding script
    // execution to an event listener gives us control over when the script
    // is executed. See http://jaubourg.net/2010/07/loading-script-as-onclick-handler-of.html
    script.event = 'onclick'
    script.htmlFor = script.id = '_reqwest_' + reqId
}

This restores the appropriate behavior in IE9 and doesn't appear to break IE8. The same code causes problems with calls to grid.json files, and commenting this code out fixes that as well.

**Comment by tmcw** [comment]
Yes, this is a known problem and is worse in IE10. This week @ansis and I are working on reevaluating reqwest & our jsonp strategy.

**Comment by mikejcorey** [comment]
Great, thanks!
